pandas/tests/io/parser/python_parser_only.py

```diff
@@ -218,6 +218,23 @@ def test_multi_char_sep_quotes(self):
                 self.read_csv(StringIO(data), sep=',,',
                               quoting=csv.QUOTE_NONE)
 
+    def test_none_delimiter(self):
+        # see gh-13374 and gh-17465
```

Comment:
can you add a 1-line comment about what is happening here.

Is this *only* in the python parser as well?

Reply:
Only the Python parser. I actually discovered the issue because I was using the built in CSV sniffer and the C parser for a while, but switched to the python engine with the pandas sniffer because it did notably better for the data files I was using.